cmd/promtool/main.go

```diff
@@ -182,6 +183,31 @@ func checkRules(t cli.Term, filename string) (int, error) {
 	return len(rules), nil
 }
 
+// CheckMetricsCmd performs a linting pass on input metrics.
+func CheckMetricsCmd(t cli.Term, args ...string) int {
+	if len(args) != 0 {
+		t.Infof("usage: curl http://localhost:9090/metrics | promtool check-metrics")
```

Comment:
Yeah.  I was trying to make the usage statement concise.

I think this should arguably accept three types of input:
- stdin pipe
- filename
- HTTP URL

I just kept it simple for now.